assets/style.css

```diff
@@ -230,10 +230,11 @@ a:visited {
  * CONTENT
  */
 
-/* .content {
-  background: #fff;
-  padding: 20px;
-} */
+.content {
+  /* background: #fff; */
+  padding: 4em 0 0 0;
```

Comment:
please use `margin-top`

Reply:
@fstreicher okay - how do I proceed now with this open branch? Close and open a new one with margin or can I resolve it directly here?